aioxmpp/security_layer.py

```diff
@@ -1376,6 +1381,12 @@ def make(
     :data:`None` while `pin_store` is not, a coroutine which returns
     :data:`False` is substituted.
 
+    `ssl_context_factory` can be a callable taking no arguments and returning
+    a :class:`OpenSSL.SSL.Context` object. If given, the factory will be used
+    to obtain an SSL context when the stream negotiates transport layer security
+    via TLS. By default, :func:`aioxmpp.security_layer.default_ssl_context` is
+    used, which should be fine for most applications.
```

Comment:
This should sound more like the hdparm manual page; we should note that you should only use another context if you know what you are doing as using a misconfigured context can disable the authentication of the peer.